lib/restforce/db/worker.rb

```diff
@@ -104,11 +113,7 @@ def track
         if tracker
           runtime = Time.now
 
-          if tracker.last_run
-            log "SYNCHRONIZING from #{tracker.last_run.iso8601}"
-          else
-            log "SYNCHRONIZING"
-          end
+          log "SYNCHRONIZING#{" from #{tracker.last_run.iso8601}" if tracker.last_run}"
```

Comment:
Dirty secret: I did it to reduce the number of lines in the class so rubocop wouldn't complain. You're right that it doesn't read as clearly. I'll work through a few changes to reduce complexity where it actually matters.